auto-value-moshi/src/main/java/com/ryanharter/auto/value/moshi/AutoValueMoshiAdapterFactoryProcessor.java

```diff
@@ -108,69 +109,88 @@
-                    return false;
-                  }
-                  break;
-              }
-              // If we got here, the class is visible. Now check the jsonAdapter method
-              ExecutableElement adapterMethod = getJsonAdapterMethod(e);
-              if (adapterMethod == null) {
+                }
+                break;
+            }
+            // If we got here, the class is visible. Now check the jsonAdapter method
+            ExecutableElement adapterMethod = getJsonAdapterMethod(e);
+            if (adapterMethod == null) {
+              return false;
+            }
+            Visibility methodVisibility = Visibility.ofElement(adapterMethod);
+            switch (methodVisibility) {
+              case PRIVATE:
                 return false;
-              }
-              Visibility methodVisibility = Visibility.ofElement(adapterMethod);
-              switch (methodVisibility) {
-                case PRIVATE:
+              case DEFAULT:
+              case PROTECTED:
+                if (!getPackage(adapterMethod).equals(packageElement)) {
                   return false;
-                case DEFAULT:
-                case PROTECTED:
-                  if (!getPackage(adapterMethod).equals(packageElement)) {
-                    return false;
-                  }
-                  break;
-              }
-              return true;
-            })
-            .collect(toList());
-
-        MoshiAdapterFactory annotation = element.getAnnotation(MoshiAdapterFactory.class);
-        boolean requestNullSafeAdapters = annotation.nullSafe();
-
-        TypeSpec jsonAdapterFactory = createJsonAdapterFactory(type,
-            applicableElements,
-            packageName,
-            adapterName,
-            requestNullSafeAdapters);
-        JavaFile file = JavaFile.builder(packageName, jsonAdapterFactory).build();
-        try {
-          file.writeTo(processingEnv.getFiler());
-        } catch (IOException e) {
-          processingEnv.getMessager()
-              .printMessage(ERROR,
-                  "Failed to write TypeAdapterFactory: " + e.getLocalizedMessage());
-        }
+                }
+                break;
+            }
+            return true;
+          })
+          .collect(toList());
+
+      MoshiAdapterFactory annotation = element.getAnnotation(MoshiAdapterFactory.class);
+      boolean requestNullSafeAdapters = annotation.nullSafe();
+
+      TypeSpec jsonAdapterFactory = createJsonAdapterFactory(type,
+          applicableElements,
+          packageName,
+          adapterName,
+          requestNullSafeAdapters);
+      JavaFile file = JavaFile.builder(packageName, jsonAdapterFactory).build();
+      try {
+        file.writeTo(processingEnv.getFiler());
+      } catch (IOException e) {
+        processingEnv.getMessager()
+            .printMessage(ERROR,
+                "Failed to write JsonAdapter.Factory: " + e.getLocalizedMessage());
```

Comment:
I did opportunistically fix this to say `JsonAdapter.Factory` instead of `TypeAdapterFactory`